recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,164 @@
+{% set version = "1.2.1" %}
+{% set build = 0 %}
+
+{% set processor = processor or 'cpu' %}
+
+{% if processor == "cpu" %}
+# prioritize CPU version via build number
+{% set build = build + 100 %}
+{% endif %}
+
+
+package:
+  name: prismatic_split
+  version: {{ version }}
+
+source:
+  url: https://github.com/prism-em/prismatic/archive/{{ version }}.tar.gz
+  sha256: 844f659575dd98999b7c4864e850a7ec735f297ab1fb6c2d2be91d7f9720fe70
+  patches:
+    - patch_apple_target_lib.txt
+
+build:
+  skip: True  # [win or py2k]
+  number: {{ build }}
+  string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+
+outputs:
+  - name: prismatic
+    build:
+      number: {{ build }}
+      skip: True  # [win or py2k]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+    requirements:
+      run:
+        - {{ pin_subpackage('prismatic_cli', exact=True) }}
+        - {{ pin_subpackage('prismatic_gui', exact=True) }}
+        - {{ pin_subpackage('pyprismatic', exact=True) }}  # [not osx]
```

Comment:
It's best practice to include the tests from the other components to make sure they're included. I've accidentally deployed empty metapackages before so it is actually useful to make sure you're doing what you think you are.

Reply:
Good point - I'll add these tests.